futures-util/src/stream/for_each_concurrent.rs

```diff
@@ -0,0 +1,106 @@
+use core::marker::Unpin;
+use core::mem::PinMut;
+use core::num::NonZeroUsize;
+use futures_core::future::Future;
+use futures_core::stream::Stream;
+use futures_core::task::{self, Poll};
+use pin_utils::{unsafe_pinned, unsafe_unpinned};
+use crate::stream::{FuturesUnordered, StreamExt};
```

Comment:
The style guide suggests to put `use crate`/`use super` on top (https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/items.md#ordering-list-import)

Reply:
Done.